csunplugged/topics/management/commands/loadtopics.py

```diff
@@ -60,6 +61,23 @@ def handle(self, *args, **options):
                     BASE_PATH
                 ).load()
 
+        if "glossary-terms" in structure_file:
+            if "glossary-folder" in structure_file:
+                glossary_folder_path = structure_file["glossary-folder"]
+            else:
+                raise MissingRequiredFieldError(
+                    structure_file_path,
+                    ["glossary-folder"],
+                    "Application Structure"
+                )
+            if glossary_folder_path is not None:
+                GlossaryTermsLoader(
+                    glossary_folder_path,
+                    structure_file["glossary-terms"],
```

Comment:
I don't think this is really necessary. I think that if we ask for the name of the folder, then we just read every md file from there as a glossary term, no need to ever list them in the structure file as we can use the file name as the key/slug.